sklearn/metrics/_plot/roc_curve.py

```diff
@@ -134,22 +225,34 @@ def plot(
         **kwargs : dict
             Keyword arguments to be passed to matplotlib's `plot`.
 
+            .. deprecated:: 1.7
+                kwargs is deprecated and will be removed in 1.9. Pass matplotlib
```

Comment:
Using "**kwargs" here instead of "kwargs" results in CI failure due to a docutils check, but I think this is fine.